static_inverse_kinematics.py

```diff
@@ -63,20 +62,35 @@ def __init__(
         self.c3d_path_file = c3d_path_file
 
         self.biorbd_model = biorbd.Model(self.biorbd_model_path)
-        self.c3d_data = C3dData(self.c3d_path_file, self.biorbd_model)
+        self.c3d = c3d(self.c3d_path_file)
 
-        self.list_model_markers = [
+        self.marker_names = [
             self.biorbd_model.markerNames()[i].to_string() for i in range(len(self.biorbd_model.markerNames()))
         ]
-        self.xp_markers = self.c3d_data.trajectories
+        self.xp_markers = self.get_marker_trajectories()
         self.nb_q = self.biorbd_model.nbQ()
-        self.nb_frames = self.c3d_data.nb_frames
+        self.nb_frames = self.c3d["parameters"]["POINT"]["FRAMES"]["value"][0]
         self.nb_markers = self.biorbd_model.nbMarkers()
 
         self.q = np.zeros((self.nb_q, self.nb_frames))
-        self.bounds_min, self.bounds_max = np.squeeze(get_range_q(self.biorbd_model))
         self.bounds = get_range_q(self.biorbd_model)
-        
+
+    def get_marker_trajectories(self) -> np.ndarray:
+        """
+        get markers trajectories
+        """
+
+        # LOAD C3D FILE
+        points = self.c3d["data"]["points"]
+        labels_markers = self.c3d["parameters"]["POINT"]["LABELS"]["value"]
+
+        # GET THE MARKERS POSITION (X, Y, Z) AT EACH POINT
+        markers = np.zeros((3, len(self.marker_names), len(points[0, 0, :])))
+
+        for i, name in enumerate(self.marker_names):
+            markers[:, i, :] = points[:3, labels_markers.index(name), :] * 1e-3
```

Comment:
find the unit in the c3d and scale the data accordingly.

Reply:
done